packages/experimental/theme/README.md

```diff
@@ -0,0 +1,60 @@
+# @fluentui-react-native/use-slots
+
+A pattern and framework for building layering components together in an efficient, pluggable, and customizable manner.
```

Comment:
building and layering?

Reply:
Doh, just copied this file (obviously) and need to update this.